pkg/cmdutil/userio/text_input.go

```diff
@@ -111,3 +112,14 @@ func (m textInputModel[V]) View() string {
 	}
 	return s.String()
 }
+
+func MaxLines(in strings.Builder) int {
```

Comment:
it's not clear what this function does from it's name.
MaxLineLength or similar feels clearer.

In addition, why no to place this function in pkg/env/list.go, where it's used? I don't see how this function is related to text_input

Reply:
to be more nit-picking I would also ask for a better name e.g. `MaxLineLength` of the function and and I would rather expect a `string` as the input type than a builder